hw/ip/otbn/doc/_index.md

```diff
@@ -246,42 +246,45 @@ GPRs are accessible from the base instruction subset, and WDRs are accessible fr
 
 ### Wide Special Purpose Registers (WSRs)
 
-In addition to the Wide Data Registers, BN instructions can also access WLEN-sized special purpose registers, short WSRs.
+OTBN has 256b Wide Special purpose Registers (WSRs).
+These are analogous to the 32b CSRs, but are used by the wide side of the processor.
```

Comment:
```suggestion
These are analogous to the 32b CSRs, but are used by the big number instruction subset.
```

(I think "wide side" isn't a term we use widely in the spec.)

Reply:
I didn't quite mean that: I meant that the values were used by the wide data path in the processor (rather than by the instructions that trigger it). But if we don't have a name for the wide side of the processor, let's go for your proposed text.